src/main/resources/application.yaml

```diff
@@ -22,7 +22,7 @@ pagination:
 #    # Change name of first-page header name
 #    firstPageName:
 #  # Possibility to change the default pagination size. If not specified all results will be returned as default
-  defaultPageSize: 20
+  defaultPageSize: 2000
```

Comment:
Wow!!! Are you sure about this? It will return 2000 elements in collection results!!! You can always add the &limit=2000 parameter to override this default value

Reply:
fixed